src/models/configs.ts

```diff
@@ -30,6 +30,5 @@ export interface ReportPortalConfig {
 }
 
 export interface AgentOptions {
-    screenshotsPath?: string;
```

Comment:
It seems you forgot to remove screenshotsPath from `src\postFactumReporter\index.ts`.

Reply:
Fixed. Thank you